packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -118,21 +124,51 @@ function WorklistDataGrid(props) {
     rowHeaderIndex,
   } = props;
 
+  // Default column size constraints
+  const defaultColumnMinimumWidth = 60;
+  const defaultColumnMaximumWidth = 300;
```

Comment:
Are these values provided by UX?

Reply:
We did not get the finalized values from UX and our product manager for this JIRA.  The values will be updated in UXPLATFORM-9147.